packages/site/src/layout/NavMenu.tsx

```diff
@@ -20,6 +20,8 @@ interface NavMenuProps {
 export const NavMenu = ({ mainContentRef }: NavMenuProps) => {
   const [open, setOpen] = useState(false);
   const { isMinimal } = useAtlantisSite();
+  const location = useLocation();
```

Comment:
Same as my comment above about grabbing just the pathname

Reply:
Done here 7d5e50a50e005c4b562e60e4202a58b610ab9ed5